projects/core/src/cart/connectors/save-cart/save-cart.adapter.ts

```diff
@@ -0,0 +1,17 @@
+import { Observable } from 'rxjs';
+import { SaveCartResult } from '../../../model/cart.model';
+
+export abstract class SaveCartAdapter {
+  /**
+   * Abstract method used to save a cart
+   *
+   * @param userId: string
+   * @param cartId: string
+   * @param fields?: {saveCartName?: string; saveCartDescription?: string}
+   */
+  abstract saveCart(
+    userId: string,
+    cartId: string,
+    fields?: { saveCartName?: string; saveCartDescription?: string }
```

Comment:
`fields` here is very confusing. In OCC, `fields` normally is used to decide the list of fields returned (BASIC, DEFAULT, FULL). In here, can we just make it 
```
 abstract saveCart(
    userId: string,
    cartId: string,
    saveCartName?: string,
    saveCartDescription?: string 
```

Reply:
Yes you are right. I thought it would cause an issue with the API but apparently not so will change.